SaintCoinach/ex.json

```diff
@@ -640,11 +742,7 @@
           "type": "repeat",
           "count": 4,
           "definition": {
-            "name": "Level",
-            "converter": {
-              "type": "link",
-              "target": "Level"
-            }
+            "name": "DiscoveryIndex"
```

Comment:
Should leave this Level link in place, some other folks are using it.

Reply:
The origonal link to "Level" isnt a correct link
The actual values are for world positions (coordinates)